examples/spark/automl_feature_scores.py

```diff
@@ -40,7 +40,7 @@
 
         oof_predictions = automl.fit_predict(
             train_data,
-            roles=roles
+            roles=dataset.roles
```

Comment:
the [same](https://github.com/sb-ai-lab/SLAMA/pull/7/files#r1223141108)

Reply:
These interfaces follows interfaces of LightAutoML, which is an "origin" of SLAMA.
Not sure if it is a good idea to change these intrefaces. Need a discussion on this matter. 
(though SLAMA change them if it is necessary for performance purposes or for convenience related to Spark specific)